### PR TITLE
Fix British English syntax and Title Case for additional Private (window) cases

### PR DESCRIPTION
**Pull request by petemill**
Fix https://github.com/brave/brave-browser/issues/6891
There were some missing capitalizations following in words _after_ "Private".

Fix https://github.com/brave/brave-browser/issues/6890
There was a syntax error in transifex translation introduced immediately prior to https://github.com/brave/brave-core/pull/3970

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [ ] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: app/resources/generated_resources_en-GB.xtb [hunk 1]
```diff
@@ -122,7 +122,7 @@
 <translation id="3021678814754966447">&amp;View Frame Source</translation>
 <translation id="9027459031423301635">Open Link in New &amp;Tab</translation>
 <translation id="1769104665586091481">Open Link in New &amp;Window</translation>
-<translation id="1432267168815937878">Open link in &amp;Private window</translation>
+<translation id="1432267168815937878">Open link in &amp;Private Window</translation>
```
Comment (mkarolin) [inline app/resources/generated_resources_en-GB.xtb]: Should `link` be capitalized as well?

File: app/resources/generated_resources_en-GB.xtb [hunk 2]
```diff
@@ -2516,12 +2516,12 @@ Keep your key file in a safe place. You will need it to create new versions of y
 <translation id="3799201711591988491">{COUNT,plural, =0{&amp;Open All}=1{&amp;Open Bookmark}other{&amp;Open All (#)}}</translation>
 <translation id="7433692219247014412">{COUNT,plural, =0{Open All in &amp;New Window}=1{Open in &amp;New Window}other{Open All (#) in &amp;New Window}}</translation>
 <translation id="2633634757060186848">{COUNT, plural,
-          =0 {Open all in &amp;Private window}
-          =1 {Open in &amp;Private window}
-          other {Open all (#) in &amp;Private window}}</translation>
+          =0 {Open all in &amp;Private Window}
+          =1 {Open in &amp;Private Window}
+          other {Open all (#) in &amp;Private Window}}</translation>
```
Comment (mkarolin) [inline app/resources/generated_resources_en-GB.xtb]: Should `all` be capitalized?

**Comment by petemill** [comment]
Thanks @mkarolin I'll update. For some reason I find these a challenge to spot!

**Comment by petemill** [comment]
tbh, I should just delete these translations so that the US English is used

File: app/resources/generated_resources_en-GB.xtb (hunk 1 above)
Comment (petemill) [reply app/resources/generated_resources_en-GB.xtb]: thanks, fixed

**Pull request merged**